orderbook/src/orderbook.rs

```diff
@@ -1,16 +1,21 @@
-use model::{Order, OrderCreation, OrderMetaData, OrderUid};
-use primitive_types::{H160, H256};
-use std::convert::TryInto;
+use model::{Order, OrderCreation, OrderMetaData};
+use serde::{Deserialize, Serialize};
 use tokio::sync::RwLock;
 
-#[derive(Debug, Eq, PartialEq)]
+#[derive(Debug, Eq, PartialEq, Serialize, Deserialize)]
 pub enum AddOrderError {
```

Comment:
It might be cleaner to remove Serialize, Deserialize here and turn `OrderPostError`'s `error_type: AddOrderError,` into `error_type: String`.  This is because the actual serialized error string is only relevant to the api (filter) and also because the orderbook might have slightly different errors than what appears in the api.
For example if we wanted to handle TooManyRequests on the filter layer, we would still need it in the enum even though the OrderBook struct never creates it.

Reply:
Yeah, I agree. I also took out the TooManyRequests error out of the struct.